floss/main.py

```diff
@@ -536,6 +535,50 @@ def main(argv=None) -> int:
 
     language = identify_language(sample=sample, static_strings=static_strings)
 
+    if language == "go":
+        enhancedStaticStrings = extract_go_strings(sample=sample, min_length=args.min_length)
+        enhancedStaticStrings = list(enhancedStaticStrings)
+
+        # Calculate String Coverage
+        original_count = 0
+        modified_count = 0
+        for original_string in static_strings:
+            original_str = original_string.string
+            for modified_string in enhancedStaticStrings:
+                modified_str = modified_string.string
+                if original_str in modified_str:
+                    modified_count += len(modified_str)
+
+            original_count += len(original_string.string)
+
+        logger.info("Percentage of strings extracted: " + str(modified_count / original_count) + "%")
+
+        interim = time()
+
+        analysis = Analysis(
+            enable_static_strings=True,
+            enable_stack_strings=False,
+            enable_tight_strings=False,
+            enable_decoded_strings=False,
+        )
```

Comment:
do we need a new analysis instance her with hard-coded values?

Reply:
for now here, ok but let's think about how we want to incorporate the language specific strings into FLOSS' general output

Reply:
I'd say we don't change main for now and do the integration once we have all the other moving parts, we're tracking the integration in #808.